<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kurs</class>
 <widget class="QMainWindow" name="kurs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>probaba</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QGroupBox" name="group_in">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="entryBut">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 253, 255);
font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Войти для редактирования</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="registrateBut">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 253, 255);
font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Зарегистрироваться</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="passwordEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Логин</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Пароль</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="loginEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(250, 250, 250);
border-color: rgb(150, 150, 150);
font: 75 10pt &quot;Myanmar Text&quot;;
border-color: rgb(226, 226, 226);</string>
      </property>
      <property name="title">
       <string>Информация о сотруднике</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Фамилия</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="FamEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Имя</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="NameEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Отчество</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="OtchEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Дата рождения</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="birthEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Должность</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="jobEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Квалификация</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="qualifEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Образование</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="eduEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Курсы повышения
квалификации</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="coursEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Сведения об аттестации</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="attEdit"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>ИНН</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="innEdit"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>СНИЛС</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="snilsEdit"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Достижения</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="progrEdit"/>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Дополнительно</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="additionsEdit"/>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(245, 245, 245);</string>
         </property>
         <property name="title">
          <string>Паспортные данные</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLineEdit" name="pasp_SerieEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="pasp_FMSEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Дата выдачи</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Кем выдан</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="pasp_NumEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Номер</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="pasp_DateEdit"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Серия</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Настройки базы</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="db_nameLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="openDBBut">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 253, 255);
font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Открыть базу</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="createDBBut">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 253, 255);
font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Создать базу</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="addPersBut">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 253, 255);
font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Добавить сотрудника</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="delPersBut">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Удалить сотрудника</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="saveBut">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(248, 253, 255);
font: 75 9pt &quot;Myanmar Text&quot;;</string>
         </property>
         <property name="text">
          <string>Сохранить изменения</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="6">
        <widget class="QListView" name="persListBut"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="HelpAction">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>registrateBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>registration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>entryBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>log_in()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createDBBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>create_db()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>664</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openDBBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>open_db()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>627</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPersBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>add_person()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>701</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>614</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>save_changes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>724</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>695</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>persListBut</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>kurs</receiver>
   <slot>select_person()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delPersBut</sender>
   <signal>clicked()</signal>
   <receiver>kurs</receiver>
   <slot>delete_person()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>758</y>
    </hint>
    <hint type="destinationlabel">
     <x>-2</x>
     <y>754</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>registration()</slot>
  <slot>log_in()</slot>
  <slot>create_db()</slot>
  <slot>open_db()</slot>
  <slot>add_person()</slot>
  <slot>select_person()</slot>
  <slot>save_changes()</slot>
  <slot>open_person()</slot>
  <slot>delete_person()</slot>
 </slots>
</ui>
